apisix/cli/ngx_tpl.lua

```diff
@@ -289,6 +289,7 @@ http {
 
     {% if enabled_plugins["limit-conn"] then %}
     lua_shared_dict plugin-limit-conn {* http.lua_shared_dict["plugin-limit-conn"] *};
+    lua_shared_dict plugin-limit-conn-redis-cluster-slot-lock {* http.lua_shared_dict["plugin-limit-conn-redis-cluster-slot-lock"] *};
```

Comment:
Why do we need to add a new share dict here?

Reply:
Because resty.lock requires a shared dictionary, and rediscluster utilizes resty.lock.